### ISSUE TITLE
Error in Insert a Null  value in record 

### ISSUE THREAD

**Issue by RouaSmisem02** [lazurdit/FluentORM#3]
**Describe the bug**
When attempting to add a record with a nullable field (e.g., StudentAddress), the application throws an error indicating that the parameter for this nullable field was not supplied, even though it is allowed to be null in the database.

**Error Message:**  
```
System.Data.SqlClient.SqlException: 'The parameterized query '(@P1_StudentName nvarchar(8),@P1_StudentAddress nvarchar(4000))' expects the parameter '@P1_StudentAddress', which was not supplied.'
```

1. In the database, ensure that the column `StudentAddress` is set to allow null values.
2. In your application, implement a method that adds a new student record without providing a value for `StudentAddress`.
3. Run the application.
4. The unhandled exception appears during execution, with the error message stating the missing `@P1_StudentAddress` parameter.

**Expected behavior**
The record with a null value for `StudentAddress` should be inserted successfully without triggering an error.

**Screenshots**
![bug](https://github.com/user-attachments/assets/8343b9b7-5f8c-4f3a-acb9-8a34d27cfc7b)

**Desktop (please complete the following information):**
 - OS: [Windows 10]

**Additional context**
This issue might be related to how null parameters are handled in the SQL query. Double-check that null values are being passed correctly in the code and ensure that the parameter is explicitly set to null when necessary.

**Comment by RouaSmisem02** [comment]
@awadsalim please can you check this bug?

**Comment by awadsalim** [comment]
@RouaSmisem02 , Thank you for the report, bug is now fixed and published in version 1.1.1 of the Nuget package, please pull latest package and/or code and test,
